src/plugins/tables/tables.js

```diff
@@ -307,6 +335,18 @@ $document.on( "click", ".wb-tables-filter [type='reset']", function( event ) {
 	return false;
 } );
 
+// converts a number string to actual number, see https://stackoverflow.com/a/42213804
+function parseLocaleNumber( stringNumber ) {
+	stringNumber = stringNumber.replace( /[^0-9\,\.\-]+/g, "" );
+	var thousandSeparator = ( 1111 ).toLocaleString().replace( /1/g, "" );
+	var decimalSeparator = ( 1.1 ).toLocaleString().replace( /1/g, "" );
+
+	return parseFloat( stringNumber
+		.replace( new RegExp( "\\" + thousandSeparator, "g" ), "" )
```

Comment:
The issue with french numbers is this line.

What this issue suppose to do is convert 25,000.00 to 25000.00, but it also changing 2 500,50 to 250050.

Any thoughts?

Reply:
This function should be removed, after further testing it causes lot of issue, maybe like:
```
var $form = $( this ),
....
$toNumber = function stringToNumber( number ) {
number = number.replace( /[^0-9\,\.\-]+/g, "" );
console.log( "number: " + number );
return parseFloat( number );
};
```
Console results are:
````
number: 38198.01
number: 25400,50
number: 34259,68
number: 47322.96
number: 25400,50
number: 35,017.12
````
Now last thing to do is to find away to deal with 35,017.12 and 34259,68.